routes/api/services/read-services.js

```diff
@@ -7,5 +7,24 @@ var router = express.Router();
 
 console.log('read-services');
 
+
+app.get("/api/services", function(req, res) {
+    // 1. Add a join to include all of users
+    db.User.findAll({}).then(function(dbService) {
+      res.json(dbService);
+    });
+  });
+
+  app.get("/api/services/:id", function(req, res) {
+    // 2; Add a join to include all of users
+    db.Author.findOne({
+      where: {
+        id: req.params.id
+      }
+    }).then(function(dbService) {
+      res.json(dbService);
+    });
+  });
+
```

Comment:
Let's add another `router.get` route for `/api/services/categories` that calls `db.ServiceCategories.findAll` and returns all the service categories as json.